server/controllers/stock/core.js

```diff
@@ -492,11 +492,12 @@ function processMultipleLots(inventories) {
   const inventoryLots = _.groupBy(inventories, 'inventory_uuid');
 
   _.map(inventoryLots, (lots) => {
-    // order lots by ascending lifetime
-    let orderedInventoryLots = _.orderBy(lots, 'lifetime', 'asc');
     // order lots also by ascending quantity
     // assuming the lot with lowest quantity is consumed first
-    orderedInventoryLots = _.orderBy(orderedInventoryLots, 'quantity', 'asc');
+    let orderedInventoryLots = _.orderBy(lots, 'quantity', 'asc');
```

Comment:
You know, you can actually combine these two calls, right?  [documentation](https://lodash.com/docs/4.17.11#orderBy).

```js
const orderedInventoryLots = _.orderBy(lots, ['quantity', 'asc'], ['lifetime', 'asc']);
```

It might be simpler to make changes in the future like this.